src/pods/toolbar/components/themeToggleButton/themeToggleButton.component.tsx

```diff
@@ -0,0 +1,16 @@
+import { useThemeContext } from '@/core/providers';
+import classes from './themeToggleButton.module.css';
+
+export const ThemeToggleButton = () => {
+  const { theme, toggleTheme } = useThemeContext();
+
+  return (
+    <button className={classes.switchTheme} onClick={toggleTheme}>
+      {theme.themeMode === 'dark' ? (
+        <svg xmlns="http://www.w3.org/2000/svg" width="1.2em" height="1.2em" viewBox="0 0 256 256"><path d="M238 96a6 6 0 0 1-6 6h-18v18a6 6 0 0 1-12 0v-18h-18a6 6 0 0 1 0-12h18V72a6 6 0 0 1 12 0v18h18a6 6 0 0 1 6 6m-94-42h10v10a6 6 0 0 0 12 0V54h10a6 6 0 0 0 0-12h-10V32a6 6 0 0 0-12 0v10h-10a6 6 0 0 0 0 12m71.25 100.28a6 6 0 0 1 1.07 6A94 94 0 1 1 95.76 39.68a6 6 0 0 1 7.94 6.79A90.11 90.11 0 0 0 192 154a90.9 90.9 0 0 0 17.53-1.7a6 6 0 0 1 5.72 1.98m-14.37 11.34q-4.42.38-8.88.38A102.12 102.12 0 0 1 90 64q0-4.45.38-8.88a82 82 0 1 0 110.5 110.5" /></svg>
```

Comment:
I think we could isolate this in icons, something like:

_./src/common/icons/light-icon.component.tsx_

```tsx
export const LightIcon = () => {
  return (
<svg xmlns="http://www.w3.org/2000/svg" width="1.2em" height="1.2em" viewBox="0 0 256 256"><path d="M238 96a6 6 0 0 1-6 6h-18v18a6 6 0 0 1-12 0v-18h-18a6 6 0 0 1 0-12h18V72a6 6 0 0 1 12 0v18h18a6 6 0 0 1 6 6m-94-42h10v10a6 6 0 0 0 12 0V54h10a6 6 0 0 0 0-12h-10V32a6 6 0 0 0-12 0v10h-10a6 6 0 0 0 0 12m71.25 100.28a6 6 0 0 1 1.07 6A94 94 0 1 1 95.76 39.68a6 6 0 0 1 7.94 6.79A90.11 90.11 0 0 0 192 154a90.9 90.9 0 0 0 17.53-1.7a6 6 0 0 1 5.72 1.98m-14.37 11.34q-4.42.38-8.88.38A102.12 102.12 0 0 1 90 64q0-4.45.38-8.88a82 82 0 1 0 110.5 110.5" /></svg>
  );
}
```

We can do the same for the DarkIcon.

And when we use it in the toggle button the code looks more cleaner

_./src/pods/components/themeToggleButton.tsx_

```tsx
export const ThemeToggleButton = () => {
  const { theme, toggleTheme } = useThemeContext();

  return (
 <button className={classes.switchTheme} onClick={toggleTheme}>
      {theme.themeMode === 'dark' ? 
        (<LightIconComponent/>)
     : 
         (<DarkIconComponent/>)
      }
  );
}
```

Add as well a footer below the icon with a label like "toggle dark/light mode"